src/sagemaker/image_uri_config/tensorflow.json

```diff
@@ -1343,6 +1344,36 @@
                     "us-west-2": "763104351884"
                 },
                 "repository": "tensorflow-inference"
+            },
+            "2.7.0": {
+                "registries": {
+                    "af-south-1": "626614931356",
+                    "ap-east-1": "871362719292",
+                    "ap-northeast-1": "763104351884",
```

Comment:
DLC is yet to release these containers. Can we remove this and introduce this later when DLC officially makes them available?